dbms/src/Client/QueryParts.h

```diff
@@ -0,0 +1,522 @@
+        {(char *)"WITH"},
+        {(char *)"TOTALS"},
+        {(char *)"HAVING"},
+        {(char *)"ORDER"},
+        {(char *)"LIMIT"},
+        {(char *)"UNION"},
+        {(char *)"AND"},
+        {(char *)"OR"},
+        {(char *)"ASC"},
+        //IN
+        {(char *)"IN"},
+        //KILL QUERY
+        {(char *)"KILL"},
+        {(char *)"QUERY"},
+        {(char *)"SYNC"},
+        {(char *)"ASYNC"},
+        {(char *)"TEST"},
+        //Table engines
+        {(char *)"TinyLog"},
+        {(char *)"Log"},
+        {(char *)"Memory"},
+        {(char *)"MergeTree"},
+        {(char *)"ReplacingMergeTree"},
+        {(char *)"SummingMergeTree"},
+        {(char *)"AggregateFunction"},
+        {(char *)"CollapsingMergeTree"},
+        {(char *)"GraphiteMergeTree"},
+        {(char *)"ReplicatedMergeTree"},
+        {(char *)"ReplicatedSummingMergeTree"},
+        {(char *)"ReplicatedReplacingMergeTree"},
+        {(char *)"ReplicatedAggregatingMergeTree"},
+        {(char *)"ReplicatedCollapsingMergeTree"},
+        {(char *)"ReplicatedGraphiteMergeTree"},
+        {(char *)"Distributed"},
+        {(char *)"Dictionary"},
+        {(char *)"Merge"},
+        {(char *)"Buffer"},
+        {(char *)"File"},
+        {(char *)"Null"},
+        {(char *)"Set"},
+        {(char *)"Join"},
+        {(char *)"View"},
+        {(char *)"MaterializedView"},
+        {(char *)"Kafka"},
+        {(char *)"MySQL"},
+        //FORMATS
+        {(char *)"TabSeparated"},
+        {(char *)"TabSeparatedRaw"},
+        {(char *)"TabSeparatedWithNames"},
+        {(char *)"TSVWithNames"},
+        {(char *)"TabSeparatedWithNamesAndTypes"},
+        {(char *)"TSVWithNamesAndTypes"},
+        {(char *)"CSV"},
+        {(char *)"CSVWithNames"},
+        {(char *)"Values"},
+        {(char *)"Vertical"},
+        {(char *)"VerticalRaw"},
+        {(char *)"JSON"},
+        {(char *)"JSONCompact"},
+        {(char *)"JSONEachRow"},
+        {(char *)"TSKV"},
+        {(char *)"Pretty"},
+        {(char *)"PrettyCompact"},
+        {(char *)"PrettyCompactMonoBlock"},
+        {(char *)"PrettyNoEscapes"},
+        {(char *)"PrettySpace"},
+        {(char *)"RowBinary"},
+        {(char *)"Native"},
+        {(char *)"XML"},
+        {(char *)"CapnProto"},
+        //TYPES
+        {(char *)"Int8"},
+        {(char *)"Int16"},
+        {(char *)"Int32"},
+        {(char *)"Int64"},
+        {(char *)"UInt8"},
+        {(char *)"UInt16"},
+        {(char *)"UInt32"},
+        {(char *)"UInt64"},
+        {(char *)"Float32"},
+        {(char *)"Float64"},
+        {(char *)"Boolean"},
+        {(char *)"String"},
+        {(char *)"FixedString"},
+        {(char *)"Date"},
+        {(char *)"DateTime"},
+        {(char *)"Enum8"},
+        {(char *)"Enum16"},
+        {(char *)"Array"},
+        {(char *)"Tuple"},
+        {(char *)"Nested"},
+        {(char *)"Expression"},
+        {(char *)"Set"},
+        //FUNCTIONS
+        {(char *)"plus"},
```

Comment:
It will be difficult to support unless we will use FunctionFactory directly (or query `system.functions` table for looser coupling).